ui/src/app/cohort-review/create-review-modal/create-review-modal.tsx

```diff
@@ -76,9 +77,8 @@ export const CreateReviewModal = withCurrentWorkspace()(
       };
     }
 
-    cancelReview() {
-      const {workspace: {id, namespace}} = this.props;
-      navigate(['workspaces', namespace, id, 'data', 'cohorts']);
+    componentDidMount(): void {
+      triggerEvent('Create Review Set', 'Click', 'Create Set Review');
```

Comment:
Nitpick: is there a reason for the different phrases "Create Review Set" and "Create Set Review"?

Reply:
I actually mistyped this. The label should be "Create Set - Review,"  which I believe uses the format seen in some other events of "{action} - {location}"